tests/CI/pilot_ci.sh

```diff
@@ -74,21 +74,20 @@ function PilotInstall(){
   default
 
   cwd=$PWD
-  cd $PILOTINSTALLDIR
-  if [ $? -ne 0 ]
-  then
-    echo -e 'ERROR: cannot change to ' $PILOTINSTALLDIR
-    return
+  if ! cd "$PILOTINSTALLDIR"; then
```

Comment:
same as above `set -e`

Reply:
I know, but `set -e` might be set by the caller of these files. There might be a reason for which you prefer not having it hardcoded in here.